library/std/src/sys/unix/ext/process.rs

```diff
@@ -171,8 +171,32 @@ pub trait ExitStatusExt {
     fn from_raw(raw: i32) -> Self;
 
     /// If the process was terminated by a signal, returns that signal.
+    ///
+    /// In other words, if `WIFSIGNALED`, this returns `WTERMSIG`.
     #[stable(feature = "rust1", since = "1.0.0")]
     fn signal(&self) -> Option<i32>;
+
+    /// If the process was terminated by a signal, says whether it dumped core.
+    #[unstable(feature = "unix_process_wait_more", issue = "none")]
+    fn core_dumped(&self) -> bool;
+
+    /// If the process was stopped by a signal, returns that signal.
+    ///
+    /// In other words, if `WIFSTOPPED`, this returns `WSTOPSIG`.  This is only possible if the status came from
+    /// a `wait` system call which was passed `WUNTRACED`, was then converted into an `ExitStatus`.
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
`#[stable(feature = "rust1", since = "1.0.0")]` is not correct since this wasn't in 1.0.0. I'd like to land these all as unstable first (like you have for `core_dumped` and `into_raw`).

Reply:
This was a c&p mistake on my part.  Will fix, thanks.